src/Symfony/Bundle/FrameworkBundle/Resources/config/form_csrf.xml

```diff
@@ -12,6 +12,7 @@
             <argument>%form.type_extension.csrf.field_name%</argument>
             <argument type="service" id="translator.default" />
             <argument>%validator.translation_domain%</argument>
+            <deprecated>"The \"%service_id%\" service is deprecated since Symfony 3.1 and will be removed in 4.0."</deprecated>
```

Comment:
CsrfExtension is not used in the fullstack framework. The framework only registers the DependencyInjectionExtension in the form registry, as this allows to keep the type extensions lazy (loading them only when necessary) instead of loading them all on first usage of the Form system (other form extensions are here when using the form component standalone, for instance in Silex)